presto-main/src/main/java/com/facebook/presto/operator/LookupOuterOperator.java

```diff
@@ -44,7 +44,7 @@
         private final List<Type> types;
         private final List<Type> probeOutputTypes;
         private final List<Type> buildOutputTypes;
-        private final Runnable onOperatorClose;
+        private final ReferenceCount referenceCount;
```

Comment:
`LookupOuterOperatorFactory` doesn't really know who else is counted under this reference count. It just counts. Moreover, the class doesn't know about `LookupSource`-s or their factories. Shh, don't tell'em.